src/test/java/com/ft/methodecontentcollectionmapper/validation/ContentCollectionValidatorTest.java

```diff
@@ -14,23 +16,84 @@
 
 @RunWith(MockitoJUnitRunner.class)
 public class ContentCollectionValidatorTest {
+
     private ContentCollectionValidator contentCollectionValidator;
 
     @Before
     public void setUp() {
         contentCollectionValidator = new ContentCollectionValidator();
     }
 
+
+    @Test(expected = ValidationException.class)
+    public void thatNullMainUuidThrowsException() {
+        final EomFile eomFile = new EomFile(null, null, null, null, null, null, null, null);
```

Comment:
There's no builder, sadly...

We could add one, but we would be creating it exclusively for testing purposes, as the EomFile is never actually instantiated by us in the production code.

Should we add one?